README.md

```diff
@@ -108,5 +110,5 @@ Your answer here.
 How is `git pull` related to `git fetch`?
 
 ```text
-Your answer here.
+Git Fetch will fetch any changes made to the master file on git hub after initially pulling the file to the local and add the changes. Git Pull would pull a new file with the changes and create a new directory.
 ```
```

Comment:
Careful here, `git fetch` will pull all of the new changes/branches locally, but not merge them in. Where as `git pull` executes a `git fetch` and `git merge` merging the changes in.